packages/ui/src/components/viewport/EditorPreview.tsx

```diff
@@ -246,6 +252,8 @@ export function EditorPreview() {
             <GridIcon />
           </ButtonCheckbox>
           <ColorPicker />
+          {/* Key is set to the state so that the component renders when the state changes */}
+          <Coordinates visibility={coords} viewState={state} viewSize={size} />
```

Comment:
To conditionally toggle a component you simply do:
```ts
{coords && <Coordinates />}
```

Reply:
Ohhhh. That's neat. I will modify the visibility and hover to just utilize that conditional rendering.